docs/how-to/h-upgrade.md

```diff
@@ -1,5 +1,7 @@
 # How to upgrade between minor versions
 
+> **Note** This feature is available on Charmed Kafka and Charmed ZooKeeper from revisions 134 and 103, respectively. Upgrade from previous versions is **not supported**, although possible (see e.g. [here](https://github.com/deusebio/kafka-pre-upgrade-patch) for a custom example).
+
```

Comment:
**suggestion (non-blocking):** From my POV, these docs are here for people coming to the product post-GA, and we will be recommending to deploy from `stable`. I think this line adds noise which won't be relevant for the majority of users going in to the future.

Recommend dropping it.